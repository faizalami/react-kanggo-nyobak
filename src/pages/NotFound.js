import Layout from '../components/Layout';
import styled from '@emotion/styled';
import { width } from '../components/utilities';

const ErrorMessage = styled.h3`
  text-align: center;
  ${width.full}
`;

export default function NotFound () {
  return (
    <Layout>
      <ErrorMessage>Not Found.</ErrorMessage>
    </Layout>
  );
}
