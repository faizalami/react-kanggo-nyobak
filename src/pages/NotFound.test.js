import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import NotFound from './NotFound';

test('NotFound matches snapshot', () => {
  expect(render(
    <MemoryRouter>
      <NotFound/>
    </MemoryRouter>,
  )).toMatchSnapshot();
});
