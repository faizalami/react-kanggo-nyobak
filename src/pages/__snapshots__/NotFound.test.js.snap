// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .emotion-0 {
  background-color: #1e293b;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  max-width: 80rem;
  margin-left: 1rem;
  margin-right: 1rem;
  padding-top: 1rem;
  padding-bottom: 1rem;
}

@media screen and (min-width: 1024px) {
  .emotion-1 {
    margin-left: auto;
    margin-right: auto;
  }
}

.emotion-2 {
  margin-top: 0rem;
  margin-bottom: 0rem;
  margin-right: 1rem;
  color: white;
  font-size: 1.5rem;
}

.emotion-3 {
  border-radius: 0.375rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1rem;
  cursor: pointer;
  padding-top: 0.5rem;
  padding-bottom: 0.5rem;
  padding-left: 1rem;
  padding-right: 1rem;
  color: white;
  background-color: #111827;
  border: 1px solid transparent;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-4 {
  border-radius: 0.375rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1rem;
  cursor: pointer;
  padding-top: 0.5rem;
  padding-bottom: 0.5rem;
  padding-left: 1rem;
  padding-right: 1rem;
  color: #4f46e5;
  background-color: transparent;
  border: 1px solid transparent;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #d1d5db;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  max-width: 80rem;
  margin-left: 1rem;
  margin-right: 1rem;
  margin-top: 2rem;
  margin-bottom: 2rem;
}

@media screen and (min-width: 1024px) {
  .emotion-5 {
    margin-left: auto;
    margin-right: auto;
  }
}

.emotion-6 {
  text-align: center;
  width: 100%;
}

<body>
    <div>
      <nav
        class="emotion-0"
      >
        <div
          class="emotion-1"
        >
          <h1
            class="emotion-2"
          >
            MyWeb
          </h1>
          <a
            class="emotion-3"
            href="/product"
          >
            Product
          </a>
          <a
            class="emotion-4"
            href="https://faizalami.github.io"
            rel="noreferrer"
            target="_blank"
          >
            About Me
          </a>
        </div>
      </nav>
      <main
        class="emotion-5"
      >
        <h3
          class="emotion-6"
        >
          Not Found.
        </h3>
      </main>
    </div>
  </body>,
  "container": .emotion-0 {
  background-color: #1e293b;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  max-width: 80rem;
  margin-left: 1rem;
  margin-right: 1rem;
  padding-top: 1rem;
  padding-bottom: 1rem;
}

@media screen and (min-width: 1024px) {
  .emotion-1 {
    margin-left: auto;
    margin-right: auto;
  }
}

.emotion-2 {
  margin-top: 0rem;
  margin-bottom: 0rem;
  margin-right: 1rem;
  color: white;
  font-size: 1.5rem;
}

.emotion-3 {
  border-radius: 0.375rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1rem;
  cursor: pointer;
  padding-top: 0.5rem;
  padding-bottom: 0.5rem;
  padding-left: 1rem;
  padding-right: 1rem;
  color: white;
  background-color: #111827;
  border: 1px solid transparent;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-4 {
  border-radius: 0.375rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1rem;
  cursor: pointer;
  padding-top: 0.5rem;
  padding-bottom: 0.5rem;
  padding-left: 1rem;
  padding-right: 1rem;
  color: #4f46e5;
  background-color: transparent;
  border: 1px solid transparent;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #d1d5db;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  max-width: 80rem;
  margin-left: 1rem;
  margin-right: 1rem;
  margin-top: 2rem;
  margin-bottom: 2rem;
}

@media screen and (min-width: 1024px) {
  .emotion-5 {
    margin-left: auto;
    margin-right: auto;
  }
}

.emotion-6 {
  text-align: center;
  width: 100%;
}

<div>
    <nav
      class="emotion-0"
    >
      <div
        class="emotion-1"
      >
        <h1
          class="emotion-2"
        >
          MyWeb
        </h1>
        <a
          class="emotion-3"
          href="/product"
        >
          Product
        </a>
        <a
          class="emotion-4"
          href="https://faizalami.github.io"
          rel="noreferrer"
          target="_blank"
        >
          About Me
        </a>
      </div>
    </nav>
    <main
      class="emotion-5"
    >
      <h3
        class="emotion-6"
      >
        Not Found.
      </h3>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
